data/data_code_80071F00.s

```diff
@@ -874,13 +874,16 @@ glabel D_800E6724
 glabel D_800E672C
 .word D_800E659C, D_800E6668
 
+# These should be floats, really
+# I believe they're the positions of the static neon signs in rainbow road
```

Comment:
Can you add todo:

I know we don't really use todo's. But this helps it be searchable later.